central/telemetry/centralclient/instance_config.go

```diff
@@ -103,8 +92,21 @@ func getInstanceConfig() (*phonehome.Config, map[string]any, error) {
 func InstanceConfig() *phonehome.Config {
 	once.Do(func() {
 		var err error
+
+		ii, _, err := store.Singleton().Get(
+			sac.WithGlobalAccessScopeChecker(context.Background(),
+				sac.AllowFixedScopes(
+					sac.AccessModeScopeKeys(storage.Access_READ_ACCESS),
+					sac.ResourceScopeKeys(resources.InstallationInfo))))
+
+		if err != nil || ii == nil {
```

Comment:
```suggestion
		if ii == nil || err != nil  {
```
I'd swap it to match the order above.